src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy

```diff
@@ -49,7 +49,7 @@ import static javax.servlet.http.HttpServletResponse.SC_OK
  */
 @Slf4j
 @CompileStatic
-@SlingServlet(methods = ['GET', 'PUT'], resourceTypes = ['twcable:grabbit/job'])
+@SlingServlet(methods = ['GET', 'PUT', 'POST'], resourceTypes = ['twcable:grabbit/job'])
```

Comment:
I used DELETE initially but changed it to POST... I thought that it is kind of an update... and I think of DELETE method as something for cleanup/removal... For me, ideal method for this would be PUT but that was used..
Open to change.. Thoughts??